crates/frontend/src/pages/experiment_list.rs

```
@@ -324,15 +324,21 @@ pub fn experiment_list() -> impl IntoView {
     // acquire tenant
     let tenant_rws = use_context::<RwSignal<Tenant>>().unwrap();
     let org_rws = use_context::<RwSignal<OrganisationId>>().unwrap();
-    let filters_rws = create_rw_signal(ExperimentListFilters::default());
     let (reset_exp_form, set_exp_form) = create_signal(0);
+    let filters_rws = use_signal_from_query(move |query_string| {
+        Query::<ExperimentListFilters>::extract_query(&query_string)
+            .map(|q| q.into_inner())
+            .unwrap_or_default()
```

Can we log when the query extraction fails , it is failing silently right now . 
```
match Query::<ExperimentListFilters>::extract_query(&query_string) {
        Ok(query) => query.into_inner(),
        Err(err) => {
            log::error!("Failed to extract query for ExperimentListFilters: {}", err);
            ExperimentListFilters::default()
        }
}
```